### PR TITLE
Bug 1221180 - search btn locator is changed in Marketplace.

### PR DESCRIPTION
**Pull request by nojunpark**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=3cbd6476ae90a475f87ac5a568eb530ac7e58dbc

File: tests/python/gaia-ui-tests/gaiatest/apps/marketplace/app.py [hunk 1]
```diff
@@ -34,7 +34,7 @@ def search(self, term):
         # This sleep seems necessary, otherwise on device we get timeout failure on display search_box sometimes, see bug 1136791
         import time
         time.sleep(10)
-
+
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/apps/marketplace/app.py]: Nit: PEP8 - Extra blank space

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=913af9d6599138c8c3ff176f94febe1bf7fa700c

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=00a4e0765be2a2dc4f78ef9b0f9c1ceb8896a765

**Pull request merged**